server/models/db/availability.js

```diff
@@ -0,0 +1,38 @@
+const { db } = require('../../lib/connection');
+
+//const getAvailabilityDetails = require('../data/AvailabilityDetails');
```

Comment:
Hi Nancy,
This is currently commented out which means that the `/data/AvailabilityDetails` is currently not used in this file. Is it possible to add a brief explanation about why the uncommented line is still kept? Personally, I would recommend uncommenting it and using it to format the returned response in line 25. Since it is an object, I would also recommend renaming the variable name to be `AvailabilityDetails` rather than `getAvailabilityDetails` to be more clear.

Reply:
> Hi Nancy,
> Good work so far. I have reviewed your code and provided some additional suggestions that I would like considered before I review further.

Hi Amy, 

Thank you for your feedback on my PR! I've reviewed your recommendations and made the changes which you've suggested! Please let me know if you have any questions/further suggestions :)

Nancy

Reply:
Hi Nancy, 
I would recommend uncommenting out this line and leveraging it to format the response returned in `getAvailabilityDetails` to be consistent with the codebase.

Reply:
Hi Amy,
Apologies for missing this comment in the previous review. 
My action: The line `const Tables = require('../../constants/tables.json');` is now being used with the response in lines 25-26 of server/models/db/availability.js  

You can now see these changes in line 5 of server/models/db/availability.js

Please confirm if this takes care of your suggestion!